in_toto/keylib.go

```diff
@@ -18,6 +18,12 @@ import (
 	"strings"
 )
 
+const (
+	rsaKeyType     string = "rsa"
+	ecdsaKeyType   string = "ecdsa"
+	ed25519KeyType string = "ed25519"
+)
+
```

Comment:
@lukpueh is this ok? Or shall I export them? I am also not sure about the name for them yet. 
I have replaced all strings for these constants now, except the ones in the tests. If you want I can touch the tests, too.

Reply:
This looks good to me as is. Would you also add constants for the schemes?

Reply:
constants for the schemes are there, as a function. Because Go can't have slice or array constants :(

Reply:
The constant schemes happened here: https://github.com/in-toto/in-toto-golang/pull/56/commits/b3da4964c48f7418004608a26c7258410fd70aa3

Reply:
I thought I use for the Schemes a slice instead of just a string, because we might want to have different schemes to choose from in the future..